includes/Modules/Sign_In_With_Google.php

```diff
@@ -45,6 +46,18 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	 */
 	const MODULE_SLUG = 'sign-in-with-google';
 
+	/**
+	 * Option name to store user ID received from Google.
+	 */
+	const GOOGLE_USER_ID_OPTION = 'googlesitekitpersistent_siwg_google_user_id';
```

Comment:
Since this is actually storing a hash, we should probably name + document it accordingly. Perhaps this (similar to on SKS)?
```suggestion
	const GOOGLE_USER_HID_OPTION = 'googlesitekitpersistent_siwg_google_user_hid';
```

Reply:
Updated.